Firestore/Source/Remote/FSTStream.m

```diff
@@ -347,6 +347,10 @@ - (void)closeWithFinalState:(FSTStreamState)finalState error:(nullable NSError *
   [self.workerDispatchQueue verifyIsCurrentQueue];
   [self cancelIdleCheck];
 
+  FSTAssert(self.delegate,
```

Comment:
Could this move up to live alongside the other assertion, just to group all the sanity checking together?

Reply:
Done.